packages/ckeditor5-engine/src/model/model.ts

```diff
@@ -689,8 +723,8 @@ export default class Model extends ObservableMixin() {
 	 * ```html
 	 * <paragraph>x</paragraph>
 	 * <blockQuote>
-	 *	<paragraph>y</paragraph>
-	 *	<heading1>fir[st</heading1>
+	 * 	<heading1>fir[st</heading1>
+	 * 	<paragraph>y</paragraph>
```

Comment:
Is this changed on purpose or the mistake?

Reply:
Mistake